src/main/java/com/amazon/opendistroforelasticsearch/sql/request/SqlRequest.java

```diff
@@ -105,4 +107,15 @@ public BoolQueryBuilder checkAndAddFilter(BoolQueryBuilder boolQuery) throws Sql
         }
         return boolQuery;
     }
+
+    public String removeSensitiveData() {
+        String query = "";
+        try {
+            query = QueryDataMask.maskData(this.sql);
+        } catch (Exception e) {
+            LogManager.getLogger().error("Caught an exception when removing sensitive data: "
+                    + e.getClass().getSimpleName());
+        }
```

Comment:
Consider using 

 ```
private static final Logger LOG = LogManager.getLogger(SqlRequest.class);

...

 LOG.error("Caught an exception when masking sensitive data ",  e)

```

Let the stack trace be logged for faster debugging, instead of Exception name.

Reply:
Sounds good, will do. Thanks!